pywhat/Data/regex.json

```diff
@@ -1,5 +1,31 @@
 [
-    {
+   {
+      "Name": "PGP Public Key",
+      "Regex": "/^(-----BEGIN PGP PUBLIC KEY BLOCK-----).*([a-zA-Z0-9//\n\/\\.\\:\\+\\ \\=]+).*(-----END PGP PUBLIC KEY BLOCK-----)$",

```

Comment:
What is '/' character at the beginning for?

Reply:
Fat finger, sorry for the inconvenience